install/kubernetes/helm/Istio/values.yaml

```diff
@@ -0,0 +1,13 @@
+Istio:
+   auth_enabled: false 
+   initializer_enabled: false
+   mixer_enabled: true
+   pilot_enabled: true
+   proxy_enabled: true
+   namespace: istio-system
+   ca_hub: default
+   ca_tag: default
+   ca_demo: false 
+   proxy_hub: default
+   proxy_tag: default
+   proxy_debug: false 
```

Comment:
I think it is safe to assume if someone is using a helm chart, they are not using the debug proxy, although others in the review may have different opinions.

Reply:
Well, master repo has proxy_debug image used by default and hardcoded, at least in the templates I found. I added a choice to select one or another.